source/content/guides/drupal/drupal-hosted-deprecated-upstream/03-prepare.md

```diff
@@ -1,27 +1,27 @@
 ---
-title: Migrate a Site That Was Created Using a Deprecated Upstream to Drupal 9
```

Comment:
@stovak , contains multiple references to D8. Should it say “prior version” instead?  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-hosted-deprecated-upstream/prepare